ItemPool.py

```diff
@@ -210,16 +210,16 @@
 }
 
 trade_items = OrderedDict([
-    ("pocket_egg",   "Pocket Egg"),
-    ("pocket_cucco", "Pocket Cucco"),
-    ("cojiro",       "Cojiro"),
-    ("odd_mushroom", "Odd Mushroom"),
-    ("poachers_saw", "Poachers Saw"),
-    ("broken_sword", "Broken Sword"),
-    ("prescription", "Prescription"),
-    ("eyeball_frog", "Eyeball Frog"),
-    ("eyedrops",     "Eyedrops"),
-    ("claim_check",  "Claim Check"),
+    ("Pocket Egg",   "Pocket Egg"),
+    ("Pocket Cucco", "Pocket Cucco"),
+    ("Cojiro",       "Cojiro"),
+    ("Odd Mushroom", "Odd Mushroom"),
+    ("Poachers Saw", "Poachers Saw"),
+    ("Broken Sword", "Broken Sword"),
+    ("Prescription", "Prescription"),
+    ("Eyeball Frog", "Eyeball Frog"),
+    ("Eyedrops",     "Eyedrops"),
+    ("Claim Check",  "Claim Check"),
```

Comment:
It doesn't seem necessary for this to be a dict anymore

Reply:
Yeah, I was lazy at the time. It's no longer a dict now.